node_modules/react-chatbox-component/dist/style.css

```diff
@@ -0,0 +1,199 @@
+@import 'https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css';
+@import 'https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css';
+@import url('https://fonts.googleapis.com/css?family=Abril+Fatface&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```

Comment:
So for this we are using bootstrap lol? Is there a reason for this?

Reply:
The bootstrap was leftover from a part we didn't need actually, but we just took that out.